sdk/src/main/java/io/opentelemetry/sdk/trace/config/TraceConfig.java

```diff
@@ -34,39 +39,66 @@
  * Builder} instance, then use the {@link
  * io.opentelemetry.sdk.trace.TracerSdkProvider#updateActiveTraceConfig(TraceConfig)} with the
  * resulting TraceConfig instance.
+ *
+ * <p>Configuration options for {@link TraceConfig} can be read from system properties, environment
+ * variables, or {@link java.util.Properties} objects.
+ *
+ * <p>For system properties and {@link java.util.Properties} objects, {@link TraceConfig} will look
+ * for the following names:
+ *
+ * <ul>
+ *   <li>{@code otel.config.sampler.probability}: to set the global default sampler for traces.
+ *   <li>{@code otel.config.max.attrs}: to set the global default max number of attributes per
+ *       {@link Span}.
+ *   <li>{@code otel.config.max.events}: to set the global default max number of {@link Event}s per
+ *       {@link Span}.
+ *   <li>{@code otel.config.max.links}: to set the global default max number of {@link Link} entries
+ *       per {@link Span}.
+ *   <li>{@code otel.config.max.event.attrs}: to set the global default max number of attributes per
+ *       {@link Event}.
+ *   <li>{@code otel.config.max.link.attrs}: to set the global default max number of attributes per
+ *       {@link Link}.
+ * </ul>
+ *
+ * <p>For environment variables, {@link TraceConfig} will look for the following names:
+ *
+ * <ul>
+ *   <li>{@code OTEL_CONFIG_SAMPLER_PROBABILITY}: to set the global default sampler for traces.
+ *   <li>{@code OTEL_CONFIG_MAX_ATTRS}: to set the global default max number of attributes per
+ *       {@link Span}.
+ *   <li>{@code OTEL_CONFIG_MAX_EVENTS}: to set the global default max number of {@link Event}s per
+ *       {@link Span}.
+ *   <li>{@code OTEL_CONFIG_MAX_LINKS}: to set the global default max number of {@link Link} entries
+ *       per {@link Span}.
+ *   <li>{@code OTEL_CONFIG_MAX_EVENT_ATTRS}: to set the global default max number of attributes per
+ *       {@link Event}.
+ *   <li>{@code OTEL_CONFIG_MAX_LINK_ATTRS}: to set the global default max number of attributes per
+ *       {@link Link}.
+ * </ul>
  */
 @AutoValue
 @Immutable
 public abstract class TraceConfig {
-  // These values are the default values for all the global parameters.
-  // TODO: decide which default sampler to use
```

Comment:
Please, open an issue or leave it in otherwise, we will forget of this detail ;)